experiment/manage_release_versions.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" This script rotates the version strings in various below listed config files.
+Run this script when you want to move the jobs to the new support cadence.
+
+Usage:
+    1. update the Version Class below with the appropriate versions.
+    2. bazel run //experiment:manage_release_versions
+    3. for some config files you may need to make more manual changes. These
+    are described as comments before config files are defined below.
+
+"""
+
+import re
+import collections
+
+# Probably should just read out of a config file in the future.
+
+class Version(object):
+    STABLE1 = "1.13"
```

Comment:
We should also have one for current version.

Reply:
Do we not represent current version as master? Should I call it stable0
Are you suggesting something like STABLE0 = "master"

Reply:
I think we don't really need to touch the master jobs, I was thinking more like CURRENT = 1.14 this way we setup jobs for the new release branch.